src/assets/ManifestAsset/index.js

```diff
@@ -0,0 +1,11 @@
+const isWebExtensionManifest = require('../../utils/isWebExtensionManifest');
+
+function resolveManifestAsset(filename) {
+  if (isWebExtensionManifest(filename)) {
+    return require('./WebExtensionManifestAsset');
+  }
+
+  return require('./WebManifestAsset');
```

Comment:
Alternatively, we can just chuck these two classes (`WebExtensionManifestAsset` and `WebManifestAsset`) into this one file and export them all (`WebManifestAsset` is still being referenced individually by the parser for `.webmanifest` file ext).